### DISCUSSION TITLE
Behaviour of "share" button on main page of repository

### DISCUSSION THREAD

**Discussion by Sergisa** [community/community#81379] [Mobile]
Now "share" button makes link of main branch of current repository. I think it will be very useful to make "share" button preparing a link of currently chosen repository brunch.
Otherwise there is no way to share a "branch link"

**Comment by kangarruu** [comment]
@Sergisa thanks for your feedback! I'll share your request with the team so we could investigate adding this in the future. While it's not exactly what you're asking for, perhaps using the share button from the repository files screen (Repository -> Code) would work for you in the meantime? The link generated from there includes the current branch. Hope that helps!

**Reply by Sergisa** [reply]
Yeah, really, I've checked it right now.
Really: when I enter a 'files list' activity I have an ability to enter the "share" process with a clean link of current branch.

But nevertheless I'd be pleasent if your developers command could consider the issue. It seems to me that my offer can be useful for users😁

**Reply by kangarruu** [reply]
I've opened an internal issue to track this 👍🏻  Thanks again for taking the time to share your suggestion!
